docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,126 @@
+# Converting To WinUI3 Using C# Conversion Analyzers
+**Note: `Microsoft.WinUI.Convert` NuGet Package is Currently Unavailable During Prerelease**
+
+## Background
+
+The purpose of the Microsoft.WinUI.Convert Nuget Package is to create a porting solution to allow Developers to convert WinUI2 projects to the new WinUI3 format.
+
+- WinUI is a native user experience (UX) framework for both Windows Desktop and UWP applications. WinUI ships as part of the Windows OS. 
+[More on WinUI](https://microsoft.github.io/microsoft-ui-xaml/), [Docs](https://docs.microsoft.com/en-us/windows/apps/winui/)
+- WinUI3 is the next version of WinUI. It runs on the native Windows 10 UI platform and supports both Windows Desktop and UWP apps. WinUI3 ships as a NuGet package.
+[More on WinUI3](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/)
+
+Converting an existing WinUI C# App to WinUI3 requires some changes to the C# source code. Most notably namespace changes from Windows.UI.* to Microsoft.UI.*
+
+This porting assistance is provided in the form of Code Analyzers and Code Fixes. 
+- A Code Analyzer extends the Visual Studio experience and provides on the fly code inspections for C#. For most inspections these Code Analyzers provide light bulb quick fixes that can be applied to users code in the form of a Code Fix. 
+- A Code Fix consumes the diagnostics created by the Code Analyzer and modifies the C# file in-line. Users may choose to apply Code Fixes one at a time or resolve all instances of a Code Fix across a File, Project, or Solution.
+
+### What The Conversion Analyzers Do:
+Analyze/Convert C# files
+
+- Updates Namespaces for Xaml Types from `Windows.*` to `Microsoft.*`
```

Comment:
Not just Xaml types, for example Composition types too.

Reply:
Reworded